pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -550,7 +566,34 @@ func (computeCfg *computeServiceConfig) processAndBuildAGSgView(
 	return addressGroupSgEnforcedView, err
 }
 
-// processAndBuildAGSgView creates synchronization content for AdressGroup SG.
+// isValidAtSg finds if an ASG is a valid nephe created AppliedTo SG and belong to same RG as VNET.
+func (computeCfg *computeServiceConfig) isValidAtSg(asgID string, vnetID string) (string, error) {
```

Comment:
use AT in caps, or isValidAppliedToSG.. I would prefer later

Reply:
Yes done